core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.template.html

```diff
@@ -28,7 +28,8 @@ <h3>Upload Thumbnail</h3>
     </div>
     <div ng-show="uploadedImage">
       <div ng-if="allowedBgColors">
-        <div>Choose background color:</div>
+        <div ng-if="allowedBgColors.length > 1">Choose background color:</div>
+        <div ng-if="allowedBgColors.length === 1">This thumbnail will have a default background color:</div>
```

Comment:
When I tried to upload, I don't remember seeing the stuff about transparent background, etc. Shouldn't that information be there (and in all other dialogs)?

Reply:
The guidelines does talk about having a transparent background. It shows up before an image is uploaded:
![guidelines_b4_upload](https://user-images.githubusercontent.com/11008603/80908314-7ba22700-8d3c-11ea-90d3-5143da692eed.png)